tests/integration/test_catalogs.py

```diff
@@ -37,6 +37,152 @@ async def test_catalogs(client: _TestClient, mocker: MockFixture) -> None:
+    ), f"For header-value '{header_value}', content-Type in response-header should be text/turtle."  # noqa: B950
+
+    header_value = "*/*,text/plain"
+    headers = MultiDict([(hdrs.ACCEPT, header_value)])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status, f"'{header_value}' failed"
+    assert (
+        "text/turtle; charset=utf-8" == response.headers["Content-Type"]
+    ), f"For header-value '{header_value}', content-Type in response-header should be text/turtle."  # noqa: B950
+
+    header_value = "application/json,application/*,*/*"
+    headers = MultiDict([(hdrs.ACCEPT, header_value)])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status, f"'{header_value}' failed"
+    assert (
+        "application/ld+json; charset=utf-8" == response.headers["Content-Type"]
+    ), f"For header-value '{header_value}', content-Type in response-header should be application/ld+json."  # noqa: B950
+
+    header_value = "*/*;q=0.8,text/plain,application/signed-exchange;q=0.9,application/ld+json"  # noqa: B950
+    headers = MultiDict([(hdrs.ACCEPT, header_value)])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status, f"'{header_value}' failed"
+    assert (
+        "application/ld+json; charset=utf-8" == response.headers["Content-Type"]
+    ), f"For header-value '{header_value}', content-Type in response-header should be application/ld+json."  # noqa: B950
+
+    header_value = "*/*;q=0.8;v=b3,text/plain,application/signed-exchange;v=b3;q=0.9,application/ld+json"  # noqa: B950
+    headers = MultiDict([(hdrs.ACCEPT, header_value)])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status, f"'{header_value}' failed"
+    assert (
+        "application/ld+json; charset=utf-8" == response.headers["Content-Type"]
+    ), f"For header-value '{header_value}', content-Type in response-header should be application/ld+json."  # noqa: B950
+
+    header_value = "application/ld+json;v=b3,*/*"
+    headers = MultiDict([(hdrs.ACCEPT, header_value)])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status, f"'{header_value}' failed"
+    assert (
+        "application/ld+json; charset=utf-8" == response.headers["Content-Type"]
+    ), f"For header-value '{header_value}', content-Type in response-header should be application/ld+json."  # noqa: B950
+
+    header_value = "*/*,text/*,application/ld+json"
+    headers = MultiDict([(hdrs.ACCEPT, header_value)])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status, f"'{header_value}' failed"
+    assert (
+        "application/ld+json; charset=utf-8" == response.headers["Content-Type"]
+    ), f"For header-value '{header_value}', content-Type in response-header should be application/ld+json."  # noqa: B950
+
+
```

Comment:
Er redd du gir litt for mye bonus til `x/x` fremfor `x/*`.
```suggestion
    header_value = "application/*, text/turtle;q=0.2"
    headers = MultiDict([(hdrs.ACCEPT, header_value)])
    response = await client.get("/catalogs", headers=headers)
    assert 200 == response.status, f"'{header_value}' failed"
    assert (
        "application/ld+json; charset=utf-8" == response.headers["Content-Type"]
    ), f"For header-value '{header_value}', content-Type in response-header should be application/ld+json."  # noqa: B950
```

Reply:
Fiksa i 36f5295